packages/react-components/react-table/src/index.ts

```diff
@@ -98,6 +98,7 @@ export {
   useTableSelectionCell_unstable,
   renderTableSelectionCell_unstable,
   tableSelectionCellClassNames,
+  CELL_WIDTH as TableSelectionCellWidth,
```

Comment:
This is just personal opinion, but I think that the virtualized data grid could be responsible for hard coding this constant itself.

In my 'ideal world' thinking, people who are recomposing our components should be aware of its internals, and to know about the width of the selection cell to offset should be 'known'.

Reply:
makes sense, I'll remove this export